exercises/resistor-colors/description.md

```diff
@@ -0,0 +1,20 @@
+If you want to build something using a Raspberry Pi, you'll probably use _resistors_. For this exercise, you need to know two things about them:
+
+* Each resistor has a resistance value.
+* Resistors are small - so small in fact that if you printed the resistance value on them, it would be hard to read.
+To get around this problem, manufacturers print color-coded bands onto the resistors to denote their resistance values. Each band acts as a digit of a number. For example, if they printed a brown strip (value 1) followed by a green strip (value 5), it would translate to the number 15.
```

Comment:
I think I pointed this out earlier.  But I see that color bands are also called strips.  Why not just call them bands throughout?
```suggestion
To get around this problem, manufacturers print color-coded bands onto the resistors to denote their resistance values. Each band acts as a digit of a number. For example, if they printed a brown band (value 1) followed by a green band (value 5), it would translate to the number 15.
```

Reply:
@F3PiX what do you think about this suggestion?

Reply:
As @rpottsoh's approved the PR without it, let's keep it as is.

Reply:
@F3PiX I don't see @rpottsoh as having approved though.

Reply:
Ooh! @rpottsoh ? Am I jumping to conclusions?